grails-app/views/stockMovement/show.gsp

```diff
@@ -548,6 +554,13 @@
                             <warehouse:message code="documents.label" default="Documents"/>
                         </a>
                     </li>
+                    <g:if test="${stockMovement?.requisition}">
+                        <li>
+                            <a href="${request.contextPath}/stockMovement/comments/${stockMovement?.id}">
```

Comment:
can't we just use `<g:link>` with proper params here?

Reply:
I did not give it much attention and copy pasted the lines from above which use anchor tags.
I can refactor that.